python/test/unit/language/test_core.py

```diff
@@ -2743,9 +2743,6 @@ def test_dot(M, N, K, num_warps, col_a, col_b, epilogue, allow_tf32, in_dtype, o
             "numpy.dot with int8 inputs will overflow while tl.dot doesn't because MMA instruction's accumulator is 32-bit"
         )
 
-    if (M, N, K, num_warps) in [(128, 256, 32, 8)]:
```

Comment:
Is this pass enabled by default? Do we still need to export OPTIMIZE_EPILOGUE=1 ?

Reply:
yes, it is enabled: https://github.com/openai/triton/blob/main/third_party/amd/backend/compiler.py#L133C28-L133C49